packages/asset-swapper/src/utils/market_operation_utils/fills.ts

```diff
@@ -150,6 +163,54 @@ function dexQuotesToPaths(
     return paths;
 }
 
+function rfqtIndicativeQuotesToPath(
```

Comment:
Why can't RFQT orders just be passed into `nativeOrdersToPath()`? Making them a separate source creates a lot of unnecessary work for the optimizer.

Reply:
They can. I was just hacking my way through it to get it under test, and then iterate on it.  I through out all of the stuff you're referring to.  See https://github.com/0xProject/0x-monorepo/pull/2555/commits/d6d4d29257c4cb0810c280f8edc33939f0e6092e